Purpose: Disable the SUSE flag.
Fixes:   None, made as part of Debian porting.
Authur:  Kel Modderman
Status:  Debian specific, showed to upstream.
---
--- a/startpar/Makefile	2012-01-12 10:07:56.546348373 +0000
+++ b/startpar/Makefile	2012-01-12 10:08:42.259013386 +0000
@@ -1,5 +1,5 @@
 VERSION = 0.58
-ISSUSE	= -DSUSE
+ISSUSE	= -DNOTSUSE
 
 INSTALL		= install -m 755
 INSTALL_DATA	= install -m 644
